docker/docker-init.sh

```diff
@@ -59,3 +60,38 @@ if [ "$SUPERSET_LOAD_EXAMPLES" = "yes" ]; then
     superset load_examples
     echo_step "4" "Complete" "Loading examples"
 fi
+}
+
+with_cypress(){
+    # Configure environment
+    echo_step "1" "Starting" "Configuring environment for Cypress"
+    sed -i '/ENABLE_REACT_CRUD_VIEWS = False/c\ENABLE_REACT_CRUD_VIEWS = True' /app/superset/config.py
+    sed -i '/ENABLE_REACT_CRUD_VIEWS = os.environ.get("ENABLE_REACT_CRUD_VIEWS", False)/c\ENABLE_REACT_CRUD_VIEWS = True' /app/tests/superset_test_config.py
+    export SUPERSET_CONFIG=${SUPERSET_CONFIG}
+    echo_step "1" "Complete" "Configuring environment for Cypress"
+
+    # Initialize the database (drop and recreate tables before)
+    echo_step "2" "Starting" "Applying DB migrations"
+    superset drop_db --yes
```

Comment:
this is a small nit, but can we make the flag `--force` instead? For someone who uses this command outside of this scope, `--force` sounds more destructive and might make them pause before using it.